windows_create_package.bat

```diff
@@ -35,6 +35,12 @@ if !errorlevel! neq 0 exit /b !errorlevel!
 copy C:\%output_user_root%\openvino\runtime\bin\intel64\Release\*.dll dist\windows\ovms
 if !errorlevel! neq 0 exit /b !errorlevel!
 
+:: Include self-contained python with dependencies required by LLM pipelines
+xcopy C:\opt\ovms-python-3.9.6-embed dist\windows\ovms\python /E /I /H
```

Comment:
We need to decide which python to use.
I recommend using on - this one  for example and chenge python39 and paths for it in: windows_install_build_dependencies.bat and build script.